internal/gps/source_manager.go

```diff
@@ -292,6 +292,11 @@ func NewSourceManager(c SourceManagerConfig) (*SourceMgr, error) {
 	return sm, nil
 }
 
+// Cachedir is a getter for the cachedir location.
```

Comment:
What about `Cachedir returns the location of the cache directory.`?

Reply:
Sure @jmank88. I'll wait for some more feedback and do it along with the other requested changes(if any).

Reply:
yep, i agree with @jmank88's suggestion.